includes/RestApi/FlowController.php

```diff
@@ -79,7 +89,7 @@ public function get_onboarding_flow_data( \WP_REST_Request $request ) {
 	 * @return \WP_REST_Response|\WP_Error
 	 */
 	public function save_onboarding_flow_data( \WP_REST_Request $request ) {
-		$flow_data = array();
+		$default_flow_data = Flows::get_data();
```

Comment:
```suggestion
		$default_flow_data = FlowService::get_default_flow_data();
```
If I am not wrong this is the right function to call for getting the default flow data as it populates even the correct ecommerce values. Else I was getting publishing as the top priority for the commerce flow when it should be selling.